app/javascript/article-form/articleForm.jsx

```diff
@@ -409,14 +409,15 @@ export class ArticleForm extends Component {
   };
 
   switchHelpContext = (event, override = null) => {
-    const id = override || event.target.id;
-    this.setState({
-      ...this.setCommonProps({
-        helpFor: id,
-        helpPosition: event.target.getBoundingClientRect().y,
-      }),
-    });
-  };
+    if (!this.state.previewShowing) {
```

Comment:
I think this can be just an early return instead of wrapping it with an if.

Reply:
I did not think of that actually I think is a better way as well.